apiv2/db_import/tests/populate_db.py

```diff
@@ -396,6 +399,28 @@ def populate_tiltseries(session: sa.orm.Session) -> Tiltseries:
         tilting_scheme="unknown",
         data_acquisition_software="unknown",
     )
+    frame = Frame(
+        run_id=RUN1_ID,
+        id=FRAME_ID,
+        acquisition_order=0,
+        deposition_id=DEPOSITION_ID1,
+        s3_frame_path="s3://test-public-bucket/30001/RUN1/Frames/frame1",
+        https_frame_path="https://foo.com/30001/RUN1/Frames/frame1",
+    )
+    session.add(frame)
```

Comment:
nit: Could this be replaced with the `populate_existing_frames` method below?

Reply:
`populate_existing_frames` does more than what we need so I moved that function back to `test_frame_import.py`